src/blade/rules_generator.py

```diff
@@ -134,7 +136,7 @@ def generate_version_file(self):
         svn_info_array = '{'
         for idx in range(svn_info_len):
             key_with_idx = self.svn_info_map.keys()[idx]
-            svn_info_line = '"%s"' % self.svn_info_map[key_with_idx]
+            svn_info_line = json.dumps(self.svn_info_map[key_with_idx])
```

Comment:
why json? json require python 2.6
http://stackoverflow.com/questions/4000678/using-python-to-generate-a-c-string-literal-of-json